src/Microsoft.Net.Http.Headers/ContentDispositionHeaderValue.cs

```diff
@@ -69,13 +69,14 @@ public IList<NameValueHeaderValue> Parameters
 
         public StringSegment Name
         {
-            get { return GetName(NameString); }
+            get { return HeaderUtilities.RemoveQuotes(GetName(NameString)); }
```

Comment:
GetName takes a constant key. It doesn't manipulate that value, it uses it for a lookup.